app/models/journeys/adjustments/ProfitOrLoss.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.journeys.adjustments
+
+import enumeratum._
+import models.common.{Enumerable, WithName}
+
+sealed trait ProfitOrLoss extends EnumEntry {
+  override def entryName: String = toString.toLowerCase
+}
+
+object ProfitOrLoss extends Enumerable.Implicits {
```

Comment:
There is a more idiomatic (enumeratum way). Check how JourneyStatus is done:

object JourneyStatus extends Enum[JourneyStatus] with utils.PlayJsonEnum[JourneyStatus]

then you can use `  val values = findValues` and you have writes/reads for free